src/libs/ReportScrollManager/index.native.js

```diff
@@ -4,13 +4,33 @@ import React from 'react';
 // to the original ref.
```

Comment:
According to [this](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#code-patterns-to-improve)
> Platform-specific solutions should not be applied to more platforms than necessary
E.g. solutions for Android or iOS should go in specific index.android.js or index.ios.js files, not in an index.native.js file.

Since we need this fix only for iOS, let's have a `index.ios.js` file

Reply:
Updated.